Modules/Registration/RegistrationMethodsv4/test/itkSimpleImageRegistrationTest.cxx

```diff
@@ -203,6 +203,17 @@ PerformSimpleImageRegistration(int argc, char * argv[])
     ITK_TEST_SET_GET_VALUE(smoothingSigmasPerLevel, affineSimple->GetSmoothingSigmasPerLevel());
   }
 
+  typename AffineRegistrationType::RealType metricSamplingPercentage = 1.0;
```

Comment:
Is this the default? If not, how much slower does the test become with this change?

Reply:
@dzenanz Yes. Please see here:
https://github.com/InsightSoftwareConsortium/ITK/blob/d19e77480914bc51a09608a7e78dfb5bc5457844/Modules/Registration/RegistrationMethodsv4/include/itkImageRegistrationMethodv4.hxx#L129
https://github.com/InsightSoftwareConsortium/ITK/blob/d19e77480914bc51a09608a7e78dfb5bc5457844/Modules/Registration/RegistrationMethodsv4/include/itkImageRegistrationMethodv4.hxx#L1301

Resolving.